bee-test/src/rand/bool.rs

```diff
@@ -0,0 +1,9 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use rand::Rng;
+
+/// Generates a random boolean.
+pub fn rand_bool() -> bool {
+    rand::thread_rng().gen::<f64>() < 0.5
+}
```

Comment:
interesting, but why not just `rand::thread_rng().gen::<bool>()` ?

Reply:
Didn't know about that, will change, thanks